cypress/integration/userCanAddProductsToOrder.feature.js

```diff
@@ -0,0 +1,88 @@
+
+describe("User can add a product to his/her order", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/products",
+      response: "fixture:product_data.json"
+    });
+
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/orders",
+      response: "fixture:post_response.json"
+    });
+
+    cy.route({
+      method: "PUT",
+      url: "http://localhost:3000/api/orders/1",
+      response: "fixture:put_response.json"
+    });
+
+    cy.visit("http://localhost:3001");
+
+    cy.route({
+      method: 'POST',
+      url: "http://localhost:3000/api/auth/sign_in",
+      response: 'fixture:login.json',
+      headers: {
+          uid: 'user@mail.com'
+      }
+  });
+cy.get('#render-login').click();
+cy.get('#login').within(() => {
+    cy.get('#email').type('user@mail.com');
+    cy.get('#password').type('password');
+    cy.get('button').contains('Submit').click()
+});
```

Comment:
I don’t want to point out the need to format you code over and over again

Reply:
Sorry, we have been trying to be thorough with the code formatter but we must have missed this in our review before submitting this PR. We will continue to try to be more thorough in our checks.